File: dotNet/CTDemo/App_Code/RefsetMember.cs
namespace CTDemo
{

    /// <summary>
    ///  A simple Concept Attribute Value reference set member. 
    ///  This class only captures the following elements of the membership: 
    ///  <li>The reference set</li>
    ///  <li>The referenced concept - the concept belong to the reference set</li>
    ///  <li>The attribute value - a concept value attached to the membership </li>
    /// </summary>
    public class RefsetMember
    {
        private Concept refsetConcept;
        private Concept referencedConcept;


        public Concept GetRefsetConcept()
        {
            return refsetConcept;
        }

        public void SetRefsetConcept(Concept refsetConcept)
        {
            this.refsetConcept = refsetConcept;
        }

        public Concept GetReferencedConcept()
        {
            return referencedConcept;
        }

        public void SetReferencedConcept(Concept referencedConcept)
        {
            this.referencedConcept = referencedConcept;
        }
    }
}

File: java/src/main/au/gov/digitalhealth/model/RefsetMember.java
package au.gov.digitalhealth.model;

import au.gov.digitalhealth.model.Concept;

/**
 * A simple Concept Attribute Value reference set member. 
 *
 * This class only captures the following elements of the membership:<ul>
 * <li>The reference set
 * <li>The referenced concept - the concept belong to the reference set
 * <li>The attribute value - a concept value attached to the membership 
 */
public class RefsetMember {

    private Concept refsetConcept;
    
    private Concept referencedConcept;
    
    public Concept getRefsetConcept() {
        return refsetConcept;
    }

    public void setRefsetConcept(Concept refsetConcept) {
        this.refsetConcept = refsetConcept;
    }

    public Concept getReferencedConcept() {
        return referencedConcept;
    }

    public void setReferencedConcept(Concept referencedConcept) {
        this.referencedConcept = referencedConcept;
    }
   
}
